modules/persistent-storage-using-a-shared-configmap-in-a-pod.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: PROCEDURE
+
+[id="persistent-storage-using-a-shared-configmap-in-a-pod_{context}"]
+= Using a shared configMap in a pod
+
+[role="_abstract"]
+
+.Prerequisites
+
+* You created a `SharedConfigMap` object for the config map that you want to share across namespaces in the cluster.
+* You have privileges to perform the following actions:
+  * Create pods and access instances of available `sharedconfigmaps.sharedresource.openshift.io`. In other words, you can run `oc get sharedconfigmaps` and get a non-empty list back.
```

Comment:
```suggestion
  * Create pods 
  * To discover which SharedConfigMaps are available for use, you can run `oc get sharedconfigmaps` and get a non-empty list back.
```

Reply:
Updated this and similar elsewhere. Thanks.